ext/standard/filestat.c

```diff
@@ -761,10 +761,14 @@ PHPAPI void php_stat(const char *filename, size_t filename_length, int type, zva
 	zend_stat_t *stat_sb;
 	php_stream_statbuf ssb;
 	int flags = 0, rmask=S_IROTH, wmask=S_IWOTH, xmask=S_IXOTH; /* access rights defaults to other */
-	char *stat_sb_names[13] = {
+	char *stat_sb_names[] = {
 		"dev", "ino", "mode", "nlink", "uid", "gid", "rdev",
 		"size", "atime", "mtime", "ctime", "blksize", "blocks"
 	};
+	zval *stat_sb_addresses[] = {
+		&stat_dev, &stat_ino, &stat_mode, &stat_nlink, &stat_uid, &stat_gid, &stat_rdev,
+		&stat_size, &stat_atime, &stat_mtime, &stat_ctime, &stat_blksize, &stat_blocks
+	};
```

Comment:
Can you please move stat_sb_names, stat_sb_addresses and the stat_ zvals down into the FS_STAT case? It looks like they are only used there.